manifests/charts/gateways/istio-ingress/templates/meshexpansion.yaml

```diff
@@ -16,7 +16,16 @@ spec:
         name: tcp-istiod
       hosts:
         - "*"
+    {{- if or .Values.sidecarInjectorWebhook.injectionURL .Values.base.validationURL }}
```

Comment:
I think it is safe to add the port here unconditionally.

Reply:
And easier for user.

Reply:
updated